api-specs/Gateway-EE/3.9/kong-ee.yaml

```diff
@@ -13111,6 +13119,69 @@ paths:
           $ref: '#/components/responses/HTTP401Error'
       operationId: get-admins-name_or_id-workspaces
       description: Return workspaces associated with an admin.
+  '/admins/{name_or_id}/workspaces/{workspace_name_or_id}':
```

Comment:
I wasn't able to test this one (I'll try again on Monday), so I'm not sure it actually works like that